packages/v4/patternfly-docs/content/design-guidelines/demos/primary-detail/primary-detail.md

```diff
@@ -8,7 +8,10 @@ A **primary-detail layout** is an interface that shows a list of items and the c
 ## Elements
 A primary component sits in a container on the page, and detail components typically live in a slide-out [drawer](/components/drawer/react).
 
-<img src="./img/floating-drawer.png"  alt="floating drawer with overview details"  width="990"/>  
+<img src="./img/primary-detail-in-data-list.png"  alt="Image showing primary-detail component in data list"  width="1500"/>  
+
+1. **Primary component:** Contains the grouping of elements that can be selected to initiate each element’s details drawer.
+2. **Drawer component:** Contains the details of each individual element.
 
 When the page content has a white background, the drawer can be placed on the same level as the primary content with a divider line. Alternatively, the drawer can be elevated on top of the primary content. We recommend having a title in the drawer that relates back to the selected primary item. 
  
```

Comment:
I feel like we should link to each primary component mentioned in this list (ie: link to data list, data table, card view, simple list)